unittests/dojo_test_case.py

```diff
@@ -469,13 +469,17 @@ def import_scan_with_params(self, filename, scan_type='ZAP Scan', engagement=1,
         if test_title is not None:
             payload['test_title'] = test_title
 
+        if scan_date is not None:
+            payload['scan_date'] = scan_date
+        else:
+            payload['scan_date'] = str(datetime.date.today())
+
```

Comment:
I think we should also test for the scenario where scan_date is None, what used to be the default in the test cases. Not sure how anybody else uses the scan_date, but I always leave it as None because it will get defaulted to today by the serializer. If we make the change proposed here, we have defaults set in two places, if one of them changes it might lead to confusing results or missed scenario's in the tests.

Reply:
I cannot think of a possible scenario when the scan date could be None. It will always have a value because there are defaults set. The default in the test cases was actually the date 2020-06-04. I can set it to actually be None (so it relies on the default) in the test case, but I think having a default in two places (not in general, but specifically one in code, and one in test) is a good thing. If someone tries to change the default in the code, the unit tests will fail. This a good thing as it will be easier to catch someone making big changes. 

Your use of the scan_date is the common use case. Leaving it as None (not specifying it at all) will **not** affect the date of the findings at all whether they were set by the parser, or set by the default value. This is reflected in the code as well.

Reply:
Still I think it would be better to have the until test reflect what most users are doing, not sending a scan_date. IT won't change what happens today, the tests will still pass.